Fastlane/Fastfile

```diff
@@ -0,0 +1,44 @@
+require_relative 'Configuration'
+
+default_platform :ios
+
+platform :ios do
+ 
+  desc "Runs all the tests"
+  lane :unit do
+    unit_test
+  end
+
+  desc "Builds alpha configuration"
+  lane :alpha do
+    build
+  end
+
+  private_lane :unit_test do
```

Comment:
End parentheses brought inline with method call. Same comment for other two calls:
 
- https://github.com/alphagov/govuk-mobile-ios-app/pull/2/files#diff-fb2c7d64b89b76b2a315e5a7365ee1f5f55938bec1a2a766bc2d1c623bda9cb4R28)
- https://github.com/alphagov/govuk-mobile-ios-app/pull/2/files#diff-fb2c7d64b89b76b2a315e5a7365ee1f5f55938bec1a2a766bc2d1c623bda9cb4R43

```
    scan(
      scheme: configuration.app_scheme,
      devices: configuration.scan_devices,
      clean: false,
      prelaunch_simulator: true,
      derived_data_path: './build_for_unit_testing'
    )
```